api/ark/backupservice/status.go

```diff
@@ -30,12 +30,15 @@ import (
 // Deprecated: use GET method instead
 func StatusDeprecated(c *gin.Context) {
 	logger := correlationid.Logger(common.Log, c)
-	logger.Info("checking ARK deployment status")
+	logger.Debug("checking ARK deployment status")
 
 	schedulesSvc := common.GetARKService(c.Request).GetSchedulesService()
 	_, err := schedulesSvc.List()
 	if err != nil {
-		err = errors.New("backup service not deployed")
+		err = errors.WrapIf(err, "backup service not deployed")
+		if !gorm.IsRecordNotFoundError(errors.Cause(err)) {
```

Comment:
How about checking the error first, before wrapping it?

Reply:
👍